provisioning/roles/cuttlefish-app/tasks/main.yml

```diff
@@ -92,3 +92,11 @@
     owner: root
     group: root
     mode: 0744
+
+- name: Create old email archiving cronjob
+  copy:
```

Comment:
It's being added to the _user crontab_ so to check it you use the `crontab` command:

```
# List the current user's crontab
crontab -l
# Edit the current user's crontab
crontab -e
```

This [section of Ubuntu's CronHowto](https://help.ubuntu.com/community/CronHowto#Two_Other_Types_of_Crontab) should give you an idea of the differences between this and the files in `/etc/cron*`.